extension/install.rdf

```diff
@@ -4,7 +4,7 @@
 
     <em:id>personas@christopher.beard</em:id>
     <em:name>Personas Plus</em:name>
-    <em:version>1.7.6</em:version>
+    <em:version>1.7.6.1</em:version>
```

Comment:
I'd like to stick to three digit version numbers, because JPM wants semantic versioning. Can we increase it to `1.7.7.`, please?